shimmingtoolbox/optimizer/basic_lsq.py

```diff
@@ -46,14 +44,17 @@ def optimize(self, unshimmed, mask, mask_origin=(0, 0, 0), bounds=None):
 
         mx, my, mz = mask_origin
         mX, mY, mZ = mask.shape
+        mV = mX * mY * mZ
```

Comment:
it would be good to stick with the camel_case convention for variables

Reply:
can_do